src/Tizen.NUI/src/internal/Interop/Interop.Touch.cs

```diff
@@ -30,6 +30,9 @@ internal static partial class Touch
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Touch_GetTime")]
             public static extern uint GetTime(global::System.Runtime.InteropServices.HandleRef jarg1);
 
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Touch_SetTime")]
+            public static extern void SetTime(global::System.Runtime.InteropServices.HandleRef touchEvent, uint time);
```

Comment:
```suggestion
            public static extern void SetTime(global::System.Runtime.InteropServices.HandleRef touchEvent, ulong time);
```

Don't we need to use `ulong` instead of `uint`? Since csharp-binder use `unsigned long` type.